docs/products/compute/compute-instances/guides/migrate-to-different-dc/index.md

```diff
@@ -53,12 +50,8 @@ Migrating your Compute Instance to a new data center will result in a number of
 
     This same menu also appears within each individual Compute Instance's dashboard page.
 
-1. In **Migrate Linode** form, review the details of the migration and check the **Accept** box to agree to these conditions and expectations.
+3. In **Migrate Linode** form, review the details of the migration and check the **Accept** box to agree to these conditions and expectations.
 
-1. Under **Configure Migration**, select the destination region. This will be the data center that the Compute Instance is migrated to.
+1. Under **Configure Migration**, select the destination Region for the migration.
```

Comment:
"Region" should likely be lower case unless it's referring to a field label, in which case it can remain capitalized formatted as bold text.